src/main/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecute.java

```diff
@@ -390,13 +392,19 @@ public static final class Config extends AbstractBigQueryActionConfig {
             "are reserved keys and cannot be used as label keys.")
     protected String jobLabelKeyValue;
 
+    @Name(NAME_READ_TIMEOUT)
+    @Nullable
+    @Macro
+    @Description("Maximum time to wait before reading response. Default is 120 seconds")
```

Comment:
nit: `Maximum value of read timeout in seconds.`

Reply:
Modified it to be similar to the description of read timeout in GCS plugins